src/main/java/io/reactivex/rxjava3/schedulers/Schedulers.java

```diff
@@ -159,6 +162,9 @@ public static Scheduler computation() {
      * <ul>
      * <li>{@code rx3.io-keep-alive-time} (long): sets the keep-alive time of the {@code io()} Scheduler workers, default is {@link IoScheduler#KEEP_ALIVE_TIME_DEFAULT}</li>
      * <li>{@code rx3.io-priority} (int): sets the thread priority of the {@code io()} Scheduler, default is {@link Thread#NORM_PRIORITY}</li>
+     * <li>{@code rx3.io-scheduled-release} (boolean): Changes release behaviour mode for {@code #io()} Scheduler.
+     *    Enabling this guarantees that workers are not released before they finish all processing, therefore preventing deadlocks.
```

Comment:
Add a new paragraph at the end of this doc instead:

```
<p>
When the {@link Scheduler.Worker} is disposed, the underlying worker can be released to the cached worker pool in two modes:
<ul>
<li>In <em>eager</em> mode (default), the underlying worker is returned immediately to the cached worker pool
 and can be reused much quicker by operators. The drawback is that if the currently running task doesn't
respond to interruption in time or at all, this may lead to delays or deadlock with the reuse use of the
underlying worker.
</li>
<li>In <em>scheduled</em> mode (enabled via the system parameter {@code rx3.io-scheduled-release}
set to {@code true}), the underlying worker is returned to the cached worker pool only after the currently running task
has finished. This can help prevent premature reuse of the underlying worker and likely won't lead to delays or
deadlock with such reuses. The drawback is that the delay in release may lead to an excess amount of underlying
workers being created.
</li>
</ul>
```

Reply:
Changed, Please double check it's in the right place now